test/functional/feature_stats.py

```
@@ -0,0 +1,141 @@
+
+    def run(self):
+        while self.running:
+            try:
+                data, _ = self.s.recvfrom(4096)
+                messages = data.decode('utf-8').strip().split('\n')
+                for msg in messages:
+                    if msg:
+                        self.queue.put(msg)
+            except socket.timeout:
+                continue
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)